src/components/button/CdrButton.vue

```diff
@@ -21,105 +97,5 @@
   </component>
 </template>
 
-<!-- NOTE: REMOVED icon slot. use icon-left -->
-<script>
-import { defineComponent, useCssModule, computed } from 'vue';
-
-import mapClasses from '../../utils/mapClasses';
-import { buildClass, buildBooleanClass } from '../../utils/buildClass';
-import propValidator from '../../utils/propValidator';
-
-export default defineComponent({
-  name: 'CdrButton',
```

Comment:
Interesting. Now that we're not providing a name property are we getting it from the filename?

Reply:
No, `<script setup>` doesn't allow the name prop. If for some reason we end up needing it, you can declare it within a normal script block.